app/models/collection.rb

```diff
@@ -5,12 +5,15 @@ class Collection < ApplicationRecord
   has_many :fetch_months
   scope :active, -> { where(active: true) }
 
+  # Removes extra spaces
+  auto_strip_attributes :title, :druid, :wasapi_collection_id
+
   validates_presence_of :title, :druid, :embargo_months, :fetch_start_month, :wasapi_provider_account,
                         :wasapi_collection_id
   validates :active, inclusion: { in: [true, false] }
   validates :admin_policy, inclusion: { in: %w[druid:wr005wn5739 druid:yf700yh0557] }
-  validates :druid, format: { with: /druid:.+/,
-                              message: 'must begin with druid:' }
+  validates :druid, format: { with: /\Adruid:[b-df-hjkmnp-tv-z]{2}[0-9]{3}[b-df-hjkmnp-tv-z]{2}[0-9]{4}\z/,
```

Comment:
I had no idea that druids could not contain vowels!? Do `\A` and and `\z` work like `^` and `$`?

Reply:
no, `^` and `$` only match up to a new-line character. `\A` and `\z` match the whole string.